apps/server/src/modules/tldraw/tldraw.module.ts

```diff
@@ -3,14 +3,14 @@ import { ConfigModule } from '@nestjs/config';
 import { createConfigModuleOptions, DB_PASSWORD, DB_USERNAME } from '@src/config';
 import { LoggerModule } from '@src/core/logger';
 import { MikroOrmModule, MikroOrmModuleSyncOptions } from '@mikro-orm/nestjs';
-import { AuthenticationModule } from '@src/modules/authentication/authentication.module';
 import { Dictionary, IPrimaryKey } from '@mikro-orm/core';
 import { CoreModule } from '@src/core';
 import { config, TLDRAW_DB_URL } from './config';
 import { TldrawDrawing } from './entities';
 import { TldrawController } from './controller';
 import { TldrawService } from './service';
 import { TldrawBoardRepo, TldrawRepo, YMongodb } from './repo';
+import { XApiKeyStrategy } from '../authentication/strategy/x-api-key.strategy';
```

Comment:
there must be a TODO comment here, direct import of a file from another module is not normally allowed...

Reply:
With adding todo, we can merge it and fix it later

Reply:
comment added